bundle/language_en.properties

```diff
@@ -663,19 +663,39 @@ angal.hospital.thevisitinghourvaluesmustbeintherange0to24.msg
+angal.inventory.notdateinfuture.msg                                                                    = Date in future is not allowed !
+angal.inventory.noproduct.msg                                                                          = Product not found!
+angal.inventory.referencealreadyused.msg                                                               = Reference is already used.
+angal.inventory.savesucces.msg                                                                         = Inventory saved. 
+angal.inventory.error.msg                                                                              = Error during  some products's recording
+angal.inventoryrow.cannotchangethepriceofproductwithoutlot.msg                                            = You can not update the unit price of product without lot.
+angal.inventoryrow.doyoureallywanttodeletethisinventoryrow.msg                                         = Do you really want to delete this inventory row?
+angal.inventoryrow.pleaseselectonlyoneinventoryrow.msg                                                 = Please select only one inventory row.
+angal.inventoryoperation.save.error.msg                                                                = Inventory not saved
 angal.inventory.nextarrow.btn                                                                          = >
 angal.inventory.arrowprevious.btn                                                                      = <
 angal.inventory.product.col                                                                            = Products
+angal.inventory.savelotwithnullprice                                                                   = Save lot with null price ?
+angal.inventory.selectmedical                                                                          = Select Medical
 angal.inventory.status.txt                                                                             = Status
 angal.inventory.specificproduct.txt                                                                    = Specific product
 angal.inventoryrow.theorticqty.col                                                                     = Theoritic Qty
+angal.inventory.totalprice                                                                             = Total price
+angal.inventoryrow.unitprice.col                                                                       = Unit price
 angal.inventory.validated                                                                              = Validated
+angal.inventory.validate.btn                                                                           = Validate
+angal.inventory.validate.btn.key                                                                       = V
+angal.inventory.view.btn                                                                               = View
+angal.inventory.view.btn.key                                                                           = E
```

Comment:
Why not use the common `View` button constants and change the button key for `Validate` instead?